azure/table_azure_app_service_web_app.go

```diff
@@ -88,6 +88,24 @@ func tableAzureAppServiceWebApp(_ context.Context) *plugin.Table {
 				Type:        proto.ColumnType_BOOL,
 				Transform:   transform.FromField("SiteProperties.HTTPSOnly"),
 			},
+			{
+				Name:        "identity_type",
```

Comment:
@Paulami30 Since there aren't that many fields here, and identity information is easy to find, would this work if we just had an `identity` column with the JSON type?

Reply:
If we fetch the whole identity column, we do not get the data. That is why I had to fetch the keys from the object.